Automattic/jetpack
# Admin Page: Remove unused file modules-per-tab-page.jsx

Remove unused file modules-per-tab-page.jsx.


The only file using this component was removed in #5997
#### Changes proposed in this Pull Request:


* Removes _inc/client/components/module-settings/modules-per-tab-page.jsx

#### Testing instructions:

1. Standing on the jetpack directory.
2. Run `grep -r AllModuleSettings *`. (The file removed by this PR exports a named export so usages of it can be greppable.
2. Confirm that no result comes up.

#### Proposed changelog entry for your changes:

* None needed
